aws-iot-securityprofile/src/main/java/com/amazonaws/iot/securityprofile/CreateHandler.java

```diff
@@ -0,0 +1,126 @@
+package com.amazonaws.iot.securityprofile;
+
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.util.concurrent.RateLimiter;
+
+import org.apache.commons.lang3.StringUtils;
+import software.amazon.awssdk.services.iot.IotClient;
+import software.amazon.awssdk.services.iot.model.AttachSecurityProfileRequest;
+import software.amazon.awssdk.services.iot.model.CreateSecurityProfileRequest;
+import software.amazon.awssdk.services.iot.model.CreateSecurityProfileResponse;
+import software.amazon.awssdk.services.iot.model.IotException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.HandlerErrorCode;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.cloudformation.resource.IdentifierUtils;
+
+public class CreateHandler extends BaseHandler<CallbackContext> {
+
+    // Copied value from software.amazon.cloudformation.resource.IdentifierUtils
+    private static final int GENERATED_NAME_MAX_LENGTH = 40;
+    private static final int MAX_CALLS_PER_SECOND_LIMIT = 5;
+
+    private final IotClient iotClient;
+
+    public CreateHandler() {
+        iotClient = IotClient.builder().build();
+    }
+
+    @Override
+    public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+            AmazonWebServicesClientProxy proxy,
+            ResourceHandlerRequest<ResourceModel> request,
+            CallbackContext callbackContext,
+            Logger logger) {
+
+        CreateSecurityProfileRequest createRequest = translateToCreateRequest(request);
+
+        ResourceModel model = request.getDesiredResourceState();
+        if (!StringUtils.isEmpty(model.getSecurityProfileArn())) {
+            logger.log(String.format("Arn is read-only, but the caller passed %s.", model.getSecurityProfileArn()));
```

Comment:
Does CFN logger not support error/info log levels?

Reply:
It doesn't, there's an open issue for it https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/issues/225